folding/src/checker.rs

```diff
@@ -69,8 +69,8 @@ impl<C: FoldingConfig> ExtendedProvider<C> {
         witness: RelaxedWitness<C::Curve, C::Witness>,
     ) -> Self {
         let inner_provider = {
-            let instance = instance.inner_instance().inner.clone();
-            let witness = witness.inner().inner.clone();
+            let instance = instance.extended_instance.instance.clone();
```

Comment:
Will create a follow-up PR to get rid of `extended` in between.

Reply:
I agree it would be nice to have a way to "hide" the extension from the "relaxed instance", but it might (not sure, yet) to know which and how many columns have been added by quadri.